### PR TITLE
Enhanced deleting and resetting of task(s), fixes #7 and closes #9, closes #10, closes #11.

### LINKED ISSUE #7: when open a task and before click oky if someone select deleat task will delete.
If I open a new task and before click okay select delete task then a task will delete , if this happen as a mistake it isn't good 

screenshot 111
screenshot 112
screenshot 113

### LINKED ISSUE #9: When deleting a task make a window saying "Are you sure?" or add undo feature
when delete a task make a window saying "are u want to delet ? " beause if we make a mistake we can't undone

### LINKED ISSUE #10: Add keyboard shortcut to Delete task
something like OPT+D

### LINKED ISSUE #11: Add a feature to delete multiple tasks and reset multiple tasks
It is hard too delete or reset multiple tasks one by one

### PR DESCRIPTION
**Pull request by AkshayAgarwal007**

### CODE REVIEW

**Comment by AkshayAgarwal007** [comment]
I'm updating it.

File: src/TimeTrackerWindow.cpp [hunk 1]
```diff
@@ -119,19 +122,42 @@ TimeTrackerWindow::MessageReceived(BMessage* message)
...
+			int32 s = 0;
+			do	
+			{	
+				if (m_ListView->IsItemSelected(s)) {
+					TaskListItem*	temp = (TaskListItem*)m_ListView->ItemAt(s);
+					temp->ResetTime();
+					m_ListView->Deselect(s);
+				}			
+			}
+			while((s = m_ListView->CurrentSelection())>=0);
```
Comment (waddlesplash) [inline src/TimeTrackerWindow.cpp:160]: Follow Haiku coding style: while goes on previous line; { for do goes on previous line.

File: src/TimeTrackerWindow.cpp [hunk 2]
```diff
@@ -142,19 +168,27 @@ TimeTrackerWindow::MessageReceived(BMessage* message)
...
+				m_ListView->Deselect(s);
+			}			
+		}
+		while((s = m_ListView->CurrentSelection())>=0);
 		m_ListView->Invalidate();
 	}
 }
 
-
+
```
Comment (waddlesplash) [inline src/TimeTrackerWindow.cpp:191]: ?

File: src/TimeTrackerWindow.cpp (hunk 1 above)
Comment (AkshayAgarwal007) [reply src/TimeTrackerWindow.cpp:160]: while goes in the above line with a tab or without?

File: src/TimeTrackerWindow.cpp (hunk 2 above)
Comment (AkshayAgarwal007) [reply src/TimeTrackerWindow.cpp:191]: Whitespaces by mistake. Sorry for that.

**Comment by waddlesplash** [comment]
Um, don't close pull requests please; just use git push --force to update them. Thanks.

**Comment by AkshayAgarwal007** [comment]
Yes I do that. Actually I made this request from the wrong branch.

**Pull request closed without merging**